### DISCUSSION TITLE
Infinity loading times

### DISCUSSION THREAD

**Discussion by Domi911-hue** [kubilus1/autoortho#165] [Q&A]
Anytime I start XP11 with Autoortho on, when readying scenery files, it just doesn't load. Any fixes or ideas on why can this happen?

**Comment by BinaryOutlook** [comment]
Disable map type override, leave it to blank and try again, some options, such as ARC certainly takes forever to load.

**Reply by Domi911-hue** [reply]
May I ask, how can I disable the map type override? Can't seem to find anything about it on Google.

**Comment by kubilus1** [comment]
You can just set it to the empty string it was originally on, but setting
it to 'BI' will match what the original set has so that would work as well.

**Reply by Domi911-hue** [reply]
May you help me with that? Like record how to do it?
